Makefile

```diff
@@ -46,6 +46,7 @@ ifneq ($(buildx), True)
 define build
 	docker build -t apache/$(1)-$(3):$(version) \
 		--build-arg checkout_v=$(checkout) \
+		--build-arg version=$(version) \
```

Comment:
@EnableAsync Please use the upper case `VERSION` here. Thanks.

Reply:
Thanks for your kind reminder, updated.